### ISSUE TITLE
hyperlink courses to testudo or planetterp

### ISSUE THREAD

**Issue by Liam-DeVoe** [konaraddi/umd-course-vis#5]
if you see a course you don't recognize, it would be nice if there was a way to find out more about the course. Each node name could be hyperlinked to its course on either testudo or planetterp.

**Comment by konaraddi** [comment]
Good idea! That would be convenient. However I don't maintain this anymore so I'm archiving this repo
